tour/beerDEcoded.yaml

```diff
@@ -0,0 +1,188 @@
+    backdrop: true
+  - title: <b>Uploading the input data</b>
+    element: .upload-button
+    intro: >-
+      We will import fastq into the history we just created.<br><br> Click
+      <b>'Next'</b> and the tour will open Galaxy Upload Manager and take you to
+      the Upload screen.
+    position: right
+    postclick:
+      - .upload-button
+  - title: <b>Uploading the input data</b>
+    intro: Upload the data by clicking the <b>'Paste/Fetch Data'</b> button.
+  - title: <b>Uploading the input data</b>
+    content: >-
+      Load the data into your history by providing the links or choose your
+      fastq file from your computer. <br>Click on <b>'Start'</b> to upload the
+      data into your Galaxy history.
+  - title: <b>Uploading the input data</b>
+    content: >-
+      The upload may take awhile.<br><br> Hit the <b>Close</b> button when you
+      see that the files are uploaded into your history.
+  - title: <b>Uploading the input data Complete !</b>
+    content: 'Now that your data is ready, let''s use some tools.<br>'
+    backdrop: true
+  - title: <b>Assign taxonomic classifications</b>
+    content: >-
+      One of the key steps in metagenomic data analysis is to identify the taxon
+      to which the individual reads belong. Taxonomic classification tools are
+      based on microbial genome databases to identify the origin of each
+      sequence.<br>
+    backdrop: true
+  - title: <b>Taxonomic classification with Kraken2</b>
+    content: >-
+      To perform the taxonomic classification we will use Kraken2. This tool
+      uses the minimizer method to sample the k-mers (all the read’s
+      subsequences of length k) in a deterministic fashion in order to reduce
+      memory constumption and processing time. In addition, it masks
+      low-complexity sequences from reference sequences by using dustmasker.<br>
+    backdrop: true
+  - title: <b>Taxonomic classification with Kraken2</b>
+    element: .toolMenuContainer
+    intro: Available tools appear here in the <b>tool menu</b>.
+    position: right
+  - title: <b>Taxonomic classification with Kraken2</b>
+    element: '#__BVID__106'
+    content: >-
+      You can use <b>'tool search'</b> to locate tools.<br><br> Search for 
+      <b>'Kraken2'</b> and select it.<br><br> Tools may take a couple of moments
+      to load.
+    placement: right
+  - title: <b>Taxonomic classification with Kraken2</b>
+    element: .toolMenuContainer
+    intro: >-
+      Open <b>'Kraken2'</b> tool<br><br> Click <b>'Next'</b> to continue our
+      tour.
+    position: right
+  - title: <b>Assign taxonomic labels with Kraken2</b>
+    element: 'div[tour_id="use_names"]'
+    intro: Have a look at the tool's parameters of the <b>'Kraken2'</b> tool.
+    placement: right
+  - title: <b>Assign taxonomic labels with Kraken2</b>
+    intro: >-
+      Please select the following parameters:<br><br> <b>'Single or paired
```

Comment:
> Would be nice to have a short explanation for each or some of the parameters to change. It could be marked as side info, so the games do not have to read them, but if they would like to learn more they can.

I added the info about parameters: Single or paired reads, Confidence and database.